.github/PULL_REQUEST_TEMPLATE/docs.md

```diff
@@ -0,0 +1,42 @@
+---
+name: Docs
+about: Updates documentation, including specifications.
+---
+
+## Description
+
+<!-- Add a description of the changes that this PR introduces and the files that
+are the most critical to review. -->
+
+closes: #XXXX
+
+---
+
+### Author Checklist
+
+*All items are required. Please add a note to the item if the item is not applicable and
+please add links to any relevant follow up issues.*
+
+I have...
+
+- [ ] included the correct `docs:` prefix in the PR title
```

Comment:
```suggestion
- [ ] included the correct `<type>:` prefix in the PR title
```

Reply:
we should do similar change in other files

Reply:
as discussed on standup, let's keep the specific reference to `docs:` for the documentation pull request template